ietf/doc/views_draft.py

```diff
@@ -995,7 +995,7 @@ def edit_shepherd_writeup(request, name):
                     dict(
                         doc=doc,
                         type="individ"
-                        if not doc.group.type.slug or doc.group.type.slug != "ietf"
+                        if not doc.group.type.slug or doc.group.type.slug != "wg"
                         else "group",
                         stream=doc.stream.slug,
```

Comment:
"Just" a style thing, but I had to stare at lines 997-999 for way too long to decipher the syntax. If it won't fit on one line, I think it either should be parenthesized as a hint or computed into a temporary variable to make the logic readable.